src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
+  let totalRent = 0;
+
+  totalRent = DayRent * days;
+
+  if (days >= 7) {
+    totalRent = DayRent * days - 50;
```

Comment:
You can simplify the logic by removing the multiplication `DayRent * days` in the if statements since `totalRent` already holds the correct value. Just subtract the discount from `totalRent`.